databricks/koala/dask/utils.py

```diff
@@ -88,7 +88,7 @@ def wrapper(method):
 
             if len(not_supported) > 0:
                 note = ("\n        Notes\n        -----\n"
-                        "        Koala doesn't support the following argument(s).\n\n")
+                        "        Dask doesn't support the following argument(s).\n\n")
```

Comment:
@thunterdb At least this should be `Koala`? otherwise "Dask" appears in the document.